src/сomponents/Experimental/FirstType/FirstType.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import { shallow, configure } from 'enzyme';
+import { shallowToJson } from 'enzyme-to-json';
+import FirstType from './FirstType';
+
+configure({adapter: new Adapter()});
```

Comment:
adapter should be configured only once. Good practice to do it in separate jest-set-up file

Reply:
fixed